Flash Chat iOS13/Controllers/ChatViewController.swift

```diff
@@ -132,4 +133,42 @@ extension ChatViewController {
 		print(indexPath.row)
 	}
 
+	// Override to support editing the table view.
+	func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
+		if editingStyle == .delete {
+			// Delete the row from the data source
+			deleteMessage(at: indexPath.row)
+		}
+	}
+
+	func deleteMessage(at row: Int) {
+		let alert = UIAlertController(title: "Delete this message?", message: "The recipients will no longer see it!", preferredStyle: .alert)
+		let deleteAction = UIAlertAction(title: "Delete", style: .destructive) { [self] action in 
+			database.collection(Constants.FStore.threadsCollectionName).document((selectedThread?.idString)!).collection(Constants.FStore.bubblesField).getDocuments() { (querySnapshot, error) in
+				if let error = error {
+					AppDelegate.showError(error, inViewController: self)
+				} else {
+					if let documents = querySnapshot?.documents {
```

Comment:
⚒️ 
(^^ emoji for "requested change")

Practicing good, specific variable naming conventions:
`documents` has meaning within Firebase, but within our app, it would be helpful for me (and any future collaborators) to understand what this represents by its variable name. Is it a thread, or a bubble?